From d7113a432c6972557c2c3efe5c7a6025331a653a Mon Sep 17 00:00:00 2001
From: Tom White <tom@cloudera.com>
Date: Mon, 10 Dec 2012 11:28:34 +0000
Subject: [PATCH 168/795] MR1: CLOUDERA_BUILD. Revert ZooKeeper version to 3.4.2.

Reason: Regression
Ref: CDH-8916
Author: Tom White
(cherry picked from commit 55670309aa6482e4f9ade62d20ff26cea27e19f3)
(cherry picked from commit 874f8cd426327329bb7c7edaddbc3f7091f503a7)
---
 ivy/libraries.properties |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/ivy/libraries.properties b/ivy/libraries.properties
index 10b5fa5..dd851b9 100644
--- a/ivy/libraries.properties
+++ b/ivy/libraries.properties
@@ -82,4 +82,4 @@ wagon-http.version=1.0-beta-2
 xmlenc.version=0.52
 xerces.version=1.4.4
 
-zookeeper.version=3.4.4
+zookeeper.version=3.4.2
-- 
1.7.0.4

